tests/utils_tests/test_autoreload.py

```diff
@@ -124,6 +125,11 @@ def test_weakref_in_sys_module(self):
         self.addCleanup(lambda: sys.modules.pop('time_proxy', None))
         list(autoreload.iter_all_python_module_files())  # No crash.
 
+    def test_module_without_spec(self):
+        module = types.ModuleType("test_module")
```

Comment:
Single quote strings

```suggestion
        module = types.ModuleType('test_module')
```